zerver/actions/user_settings.py

```diff
@@ -150,8 +150,7 @@ def do_change_full_name(
         modified_user=user_profile,
         event_type=RealmAuditLog.USER_FULL_NAME_CHANGED,
         event_time=event_time,
-        extra_data=old_name,
-        extra_data_json={RealmAuditLog.OLD_VALUE: old_name},
+        extra_data={RealmAuditLog.OLD_VALUE: old_name},
```

Comment:
Can we set a `NEW_VALUE` on here as well?
```suggestion
        extra_data={RealmAuditLog.OLD_VALUE: old_name, RealmAuditLog.NEW_VALUE: full_name},
```
Also arguable that such a change belongs in the earlier commit, and not in this one -- up to you.

Reply:
One minor concern is that we can't backfill `RealmAuditLog.NEW_VALUE` for audit log entries prior to this change. But I think such inconsistency should be fine, except that we need to special case this in the backfilling migration to avoid overriding entries that do have NEW_VALUE populate, because double write is ineffective here.

Reply:
Yeah, the lack of historical `NEW_VALUE` is what I was attempting to [discuss earlier](https://github.com/zulip/zulip/pull/23116#discussion_r1038578336).  But I'd not thought about how that interacts with the backfill -- excellent point.

Reply:
I see, thanks for the review!